jest.config.js

```diff
@@ -1,5 +1,11 @@
+// Removes the warning that we are using an unsupported version of TypeScript.
+process.env.TS_JEST_DISABLE_VER_CHECKER = false;
+
 module.exports = {
-  transform: { "^.+\\.(t|j)sx?$": "ts-jest" },
+  transform: {
+    "^.+\\.tsx?$": "ts-jest",
+    "^.+\\.jsx?$": ["babel-jest", { configFile: "../../jest.babel.js" }],
```

Comment:
How do you feel if we keep all of the config files with `*.config.js` at the end?

Reply:
I had that at the beginning and felt too long, but it's fine. I'll change it again 👍

Reply:
Done.